demo/allegro5/main.c

```diff
@@ -115,10 +115,17 @@ int main(void)
     ctx = nk_allegro5_init(font, display, WINDOW_WIDTH, WINDOW_HEIGHT);
 
     /* style.c */
-    /*set_style(ctx, THEME_WHITE);*/
-    /*set_style(ctx, THEME_RED);*/
-    /*set_style(ctx, THEME_BLUE);*/
-    /*set_style(ctx, THEME_DARK);*/
+    #ifdef INCLUDE_STYLE
+    #ifdef STYLE_WHITE
+    set_style(ctx, THEME_WHITE);
+    #elif defined(STYLE_RED)
+    set_style(ctx, THEME_RED);
+    #elif defined(STYLE_BLUE)
+    set_style(ctx, THEME_BLUE);
+    #elif defined(STYLE_DARK)
+    set_style(ctx, THEME_DARK);
+    #endif
+    #endif
```

Comment:
Maybe add a short comment as reasoning what this ifdef is useful for to make it cleare to newcomers who will be surprised if there is any **need** to do these ifdefs to make Nuklear work properly. We shall emphasize Nuklear itself and its use does not need these ifdefs.

Perhaps something like:

`/* ease regression testing during Nuklear release process; not needed for anything else */`

(this shall be present in each demo file, of course)